cli/src/main.rs

```diff
@@ -77,24 +75,69 @@ fn run() -> Result<(), CliError> {
                 (@arg force: --force "Overwrite files if they exist")
             )
         )
-        (@subcommand cert =>
-            (about: "Generate certificates that can be used for development")
-            (@subcommand generate =>
-                (about: "Generate certificates and keys for the ca, server and client")
-                (@arg common_name: --("common-name") +takes_value
-                  "The common name that should be used in the generated cert, default localhost")
-                (@arg cert_dir: -d --("cert-dir") +takes_value
-                  "Name of the directory in which to create the certificates")
-                (@arg force: --force  conflicts_with[skip] "Overwrite files if they exist")
-                (@arg skip: --skip conflicts_with[force] "Check if files exists, generate if missing")
-            )
-        )
+    );
+
+    app = app.subcommand(
+        SubCommand::with_name("cert")
+            .about("Generate certificates that can be used for development")
+            .setting(AppSettings::SubcommandRequiredElseHelp)
+            .subcommand(
+                SubCommand::with_name("generate")
+                    .long_about(
+"This command generates certificate and keys that can be used for running the Splinter daemon with
+TLS and in insecure mode. The following files are created:
+    - client.crt
+    - client.key
+    - server.crt
+    - server.key
+    - generated_ca.pem
+    - generated_ca.key
+
+The files are generated in the location specified by --cert-dir, the SPLINTER_CERT_DIR
+environment variable, or in the default location /etc/splinter/certs/.
```

Comment:
How about moving the list of files and default value to an `after_help` section?  Doing this and slightly expanding the `about` description would mean that this `long_help` would be necessary.